emcc.py

```diff
@@ -3241,17 +3241,21 @@ def phase_embind_emit_tsd(options, in_wasm, wasm_target, memfile, js_syms):
   # Required function to trigger TS generation.
   settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE += ['$callRuntimeCallbacks']
   settings.EXPORT_ES6 = False
+  # Disable minify since the binaryen pass has not been run yet to change the
+  # import names.
+  settings.MINIFY_WASM_IMPORTED_MODULES = False
```

Comment:
This is part of another change right?

Reply:
Yeah it's parts of #20376. I'll be landing that first.